src/js/modules/scrollTopArrow.js

```diff
@@ -23,4 +24,17 @@ function moveToTop() {
 }
 
 window.addEventListener("scroll", throttledMoveToTop);
-window.addEventListener("resize", throttledMoveToTop);
\ No newline at end of file
+window.addEventListener("resize", throttledMoveToTop);
```

Comment:
too complicated
set css variable in the root, use padding-right with css variable on class `stopPageScroll` and use the same variable for `button-scroll-top` in padding-right

with js set value for css variable

when modal is closed css variable has to have value 0

Reply:
Done